apps/console/src/features/identity-providers/models/identity-provider.ts

```diff
@@ -588,6 +589,10 @@ export interface LocalAuthenticatorInterface extends CommonPluggableComponentInt
      * Details endpoint.
      */
     self?: string;
+    /**
+     * The list of tags that the authenticator can be categorized under.
+     */
+    tags?: string[];
```

Comment:
Do we have to change this interface? The concern here is that we don't get `tags` from the IdP REST API.